docs/swagger.yaml

```diff
@@ -0,0 +1,164 @@
+definitions:
+  controllers.Ping:
+    properties:
+      ping:
+        example: pong
+        type: string
+    type: object
+  controllers.Welcome:
+    properties:
+      Title:
+        example: Gaia
+        type: string
+    type: object
+externalDocs:
+  description: OpenAPI
+  url: https://swagger.io/resources/open-api/
+host: plic-back-qp6wugltyq-ew.a.run.app/
```

Comment:
ya cette erreur la sur le swagger : 
![image](https://github.com/Master-Trolls-of-IT/PLIC-back/assets/124256635/103ac8b1-d321-4c01-9670-fdfca693deb4)